### PR TITLE
bug(VIST-CPC 930): fixing bug related to date not being displayed on a new visit 

### PR DESCRIPTION
**Pull request by MichaelVacca**
JIRA: link to jira ticket: https://champlainsaintlambert.atlassian.net/jira/software/c/projects/CPC/boards/35/backlog?selectedIssue=CPC-930

## Context:
This pull request involves fixing a bug not showing the correct date when a new visit is added

## Changes
- I changed the date being passed in the request payload to be called visitDate instead of date
- Changed the formating to include the hour and minute on the front end 
## Before and After UI (Required for UI-impacting PRs)
If this is a change to the UI, include before and after screenshots to show the differences.
If this is a new UI feature, include screenshots to show reviewers what it looks like. 

## UI
Home Page
image
New Visit
image
New Visit on list page 
image

### CODE REVIEW

File: api-gateway/src/main/resources/static/scripts/visits/visits.template.html [hunk 1]
```diff
@@ -482,49 +482,100 @@ <h1></h1>
...
+                selectedTime = "";
+            }
+
+            // Access the AngularJS scope
+            let scope = angular.element(document.querySelector('.container-calendar')).scope();
+
+            // Update the chosenDate on the scope
+            scope.$apply(function() {
+                scope.$ctrl.chosenDate = date;
+            });
```
Comment (Iantomasi) [inline api-gateway/src/main/resources/static/scripts/visits/visits.template.html:541]: hey mike, great work and thanks for commenting the logic but do you think you could give me a quick explanation of what's going on here?

**Review by Iantomasi — APPROVED** [review]
Awesome stuff mike. Thrilled you fixed the bug I know it took a lot of time and effort on your part

**Review by SandwichKing978 — APPROVED** [review]
Great Job on fixing the bug, keep up the good work

File: api-gateway/src/main/resources/static/scripts/visits/visits.template.html (hunk 1 above)
Comment (MichaelVacca) [reply api-gateway/src/main/resources/static/scripts/visits/visits.template.html:541]: Ya sure, line 536 just gets the scope of the calendar (where the user selects a date)
Lines 539 to 541 just changes the chosenDate variable in the visits-controller.js file to the value chosen in the calendar.

**Pull request merged**